smriprep/workflows/outputs.py

```diff
@@ -731,6 +765,8 @@ def init_ds_surfaces_wf(
                 ds_surf.inputs.desc = 'reg'
                 if surf == 'sphere_reg_fsLR':
                     ds_surf.inputs.space = 'fsLR'
+                elif surf == 'sphere_reg_dhcpAsym':
+                    ds_surf.inputs.space = 'dhcpAsym'
```

Comment:
Are you using msmsulc to align to dhcpAsym, or is that what MCRIBs registers to by default?

Reply:
MCRIBs already produces the registration to dhcpAsym, which makes it easier to get to CIFTI space.

We aren't running msmsulc yet, mostly due to my lack of clarity of the differences between dhcpAsym / fsLR surfaces.